xontrib/bashisms.py

```diff
@@ -3,9 +3,6 @@
 import sys
 import re
 
-from prompt_toolkit.keys import Keys
-from prompt_toolkit.filters import Condition, EmacsInsertMode, ViInsertMode
-
 from xonsh.platform import ptk_shell_type
```

Comment:
```suggestion

```

Reply:
The question is, should we keep this for prompt_toolkit1 users:

```
from xonsh.platform import ptk_shell_type

@events.on_ptk_create
def custom_keybindings(bindings, **kw):
    if ptk_shell_type() != "prompt_toolkit2":
        from prompt_toolkit.keys import Keys
        from prompt_toolkit.filters import Condition, EmacsInsertMode, ViInsertMode
        
        handler = bindings.registry.add_binding

        @Condition
        def last_command_exists(cli):
            return len(__xonsh__.history) > 0

        insert_mode = ViInsertMode() | EmacsInsertMode()

        @handler(Keys.Escape, ".", filter=last_command_exists & insert_mode)
        def recall_last_arg(event):
            arg = __xonsh__.history[-1].cmd.split()[-1]
            event.current_buffer.insert_text(arg)
```

Reply:
ptk1 is deprecated, as far as I'm concerned.  The only time we should touch those files is when we strip them out.